### DISCUSSION TITLE
Allow post-merge code review

### DISCUSSION THREAD

**Discussion by talmewho-work** [community/community#70480] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Some companies switched to reviewing code post-merge (but pre-release). GitHub does not allow you to approve a pull request after is has been merged.
This feature requests asks to allow it post-merge, maybe as a setting.

**Comment by pinobatch** [comment]
This is affecting me at <https://github.com/pinobatch/240p-test-mini/pull/52>. I forgot to notify GitHub.com of my approval before I clicked Merge.

**Comment by skmirajulislam** [comment]
Ummm 🤔

**Comment by AntonioND** [comment]
A workaround for this is to install the mobile GitHub app (I use Android, I'm not sure if this will work in iPhone). The UI is different there and it lets you review the PR after it has been merged. However, yes, this is inconvenient.
